### PR TITLE
Doc 1431 sfi print and post styling output uppercase

### PR DESCRIPTION
**Pull request by GodsonLeigh**

### CODE REVIEW

File: app/generator/content/get-address.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
-const { millimetresToPoints } = require('../conversion')
+const MAX_CHARACTERS_PER_LINE = 50 // Define the maximum number of characters per line
```
Comment (samplackett) [inline app/generator/content/get-address.js:1]: Probably don't need a comment here - the variable name is plenty descriptive

File: app/generator/print-styles.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+const { millimetresToPoints } = require('./conversion')
+
+const zeroMillimeters = 0
```
Comment (samplackett) [inline app/generator/print-styles.js:3]: you're not going to be happy about this one - can we make it just something like zeroMargin to more clearly represent what this is used for?

Comment (GodsonLeigh) [reply app/generator/print-styles.js]: also done

**Review by samplackett — DISMISSED** [review]
71487bca-9c85-4c18-8b95-8a5ff9413bc7_text

File: test/unit/generator/content/get-address.test.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ describe('get address', () => {
 
     test('should return business name in uppercase and new line', () => {
       const result = getAddress(businessName, address)
-      expect(result.stack[0].text).toBe(`${businessName.toUpperCase()}\n`)
+      expect(result.stack[0].text).toBe(`${businessName.toUpperCase()}\n\n\n\n\n\n\n\n`)
```
Comment (samplackett) [inline test/unit/generator/content/get-address.test.js:16]: sandstorm-darude

**Review by samplackett — APPROVED** [review]
darude-sandstorm

**Review by RicCaw — COMMENTED** [review]
Testing passed

**Pull request merged**